uk.co.compucorp.civicrm.tasksassignments/js/test/globals.js

```diff
@@ -1,21 +1,34 @@
-(function (CRM) {
```

Comment:
This is not how it should be implemented, here is a file by @reneolivo, he worked on a task which requires Drupal var.

https://github.com/civicrm/civihr/blob/523af796ce1c3105268a9f9aa9c34b0030bbf998/org.civicrm.reqangular/src/common/services/session.js

I suggest to copy his version and then @reneolivo will resolve conflicts by overriding this file with his one.